torch/csrc/jit/tracer.h

```diff
@@ -187,18 +191,10 @@ inline void abandon() {
   setTracingState(nullptr);
 }
 
-// Pre-recorded information about the trace before we actually carry
-// out the trace
-struct PreTraceInfo {
```

Comment:
Is there really any value in removing this? I left this out when doing a tracer refactor specifically because:
1. it doesn't hurt us
2. in case we need to stash something else in the future, all the code is simply there.

Reply:
It sounds a bit like YAGNI to me, no? I feel your concern, but figured since this code has been around for a while, and you haven't had to update it, it probably won't change again in the future? I think the code is overall simpler now. But I also like having strong types like `PreTraceInfo`. Bit split